bus/client/contracts.go

```diff
@@ -12,32 +12,32 @@ import (
 )
 
 // AddContract adds the provided contract to the metadata store.
-func (c *Client) AddContract(ctx context.Context, contract rhpv2.ContractRevision, contractPrice, totalCost types.Currency, startHeight uint64, state string) (added api.ContractMetadata, err error) {
+func (c *Client) AddContract(ctx context.Context, contract rhpv2.ContractRevision, contractPrice, initialRenterFunds types.Currency, startHeight uint64, state string) (added api.ContractMetadata, err error) {
 	err = c.c.WithContext(ctx).POST(fmt.Sprintf("/contract/%s", contract.ID()), api.ContractAddRequest{
-		Contract:      contract,
-		StartHeight:   startHeight,
-		ContractPrice: contractPrice,
-		State:         state,
-		TotalCost:     totalCost,
+		Contract:           contract,
+		ContractPrice:      contractPrice,
+		InitialRenterFunds: initialRenterFunds,
+		StartHeight:        startHeight,
+		State:              state,
 	}, &added)
 	return
 }
 
 // AddRenewedContract adds the provided contract to the metadata store.
-func (c *Client) AddRenewedContract(ctx context.Context, contract rhpv2.ContractRevision, contractPrice, totalCost types.Currency, startHeight uint64, renewedFrom types.FileContractID, state string) (renewed api.ContractMetadata, err error) {
+func (c *Client) AddRenewedContract(ctx context.Context, contract rhpv2.ContractRevision, contractPrice, initialRenterFunds types.Currency, startHeight uint64, renewedFrom types.FileContractID, state string) (renewed api.ContractMetadata, err error) {
```

Comment:
Hm, I removed it but the reason I kept it is because you have to be able to do contract management yourself right. Perhaps we should add `/contract/:id/archive`?